app/models/task.rb

```diff
@@ -4,8 +4,9 @@ class Task < ApplicationRecord
   validates :deadline, presence: true
   validates :status, presence: true
   validates :priority, presence: true
-  validate :date_cannot_be_in_the_past
+  before_create :date_cannot_be_in_the_past
   enum priority: { 低: 0, 中: 1, 高: 2 }
```

Comment:
enumの値は英語で定義することで、メソッドとして利用できます。
修正が大変でしたら、次回から英語で記載いただければ大丈夫です。
```suggestion
  enum priority: { low: 0, middle: 1, high: 2 }
```